tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -947,3 +953,289 @@ def test_observability_enabled_by_default_on_cluster(self, cluster):
         if cluster._default_env:
             env_vars = cluster._default_env.env_vars
             assert env_vars.get("disable_observability") == "True"
+
+    ####################################################################################################
+    # Cluster list test
+    ####################################################################################################
+
+    @pytest.mark.level("local")
+    @pytest.mark.clustertest
+    def test_cluster_list_pythonic(self, cluster):
+        original_username = rns_client.username
+        try:
+            new_username = (
+                "test-org" if "test-org" in cluster.rns_address else original_username
```

Comment:
maybe a minor difference but its really if the top level folder is equal to "test-org" no? (if rns address is something like `/test-org/my-cluster-name`)